test/gst-va/encode/10bit/vp9.py

```diff
@@ -0,0 +1,104 @@
+###
+### Copyright (C) 2019-2020 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from .....lib import *
+from .....lib.gstreamer.va.util import *
+from .....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "10bit")
+
+@slash.requires(*have_gst_element("vavp9enc"))
+@slash.requires(*have_gst_element("vavp9dec"))
+class VP9_10EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "vp9",
+      gstencoder    = "vavp9enc",
```

Comment:
There should be `vavp9enc` and `vavp9lpenc` elements.  Need to distinguish the correct one in the LP test.  See `test/gst-va/encode/10bit/hevc.py` for reference.

Reply:
Updated